x-pack/plugins/uptime/server/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -18,27 +18,28 @@ export const syntheticsMonitor: SavedObjectsType = {
     dynamic: false,
     properties: {
       name: {
-        type: 'keyword',
+        type: 'text',
+        fielddata: true,
       },
       id: {
         type: 'keyword',
       },
       type: {
-        type: 'keyword',
+        type: 'text',
       },
       urls: {
-        type: 'keyword',
+        type: 'text',
       },
       tags: {
-        type: 'keyword',
+        type: 'text',
       },
```

Comment:
We might want to review upcoming feature requests for searching, and ensure that anything we want to search in the future has mappings.

That being said, we may end up simply converting Uptime saved objects to Synthetics saved objects in the future, and Uptime may never get full search and filter features

Reply:
> We might want to review upcoming feature requests for searching, and ensure that anything we want to search in the future has mappings.

You can update the mappings after the fact.
For this release, I'd recommend omitting all fields that don't _need_ to be searched in this release.

See https://docs.elastic.dev/kibana-dev-docs/tutorials/saved-objects#mappings for more details.